svf-llvm/lib/LLVMUtil.cpp

```diff
@@ -1040,15 +1038,17 @@ const std::string SVFValue::toString() const
     {
         rawstr << "BasicBlock: " << bb->getName() << " ";
     }
+    else if (LLVMModuleSet::getLLVMModuleSet()->getLLVMValue(this) == nullptr)
```

Comment:
This is not compatible with other else if branches. It should be `else if (const SVFInstruction* inst = SVFUtil::dyn_cast<SVFInstruction>(this))`